bokeh/models/plots.py

```diff
@@ -572,6 +533,16 @@ def border_fill(self, color):
             """)
         self.border_fill_color = color
 
+    @property
+    def title(self):
+        return self._title.text[0]
```

Comment:
This is another reason not to subclass from `Label`, if `Title.text` is just a `String` then this sort of thing would not be needed